src/nvim/highlight_group.c

```diff
@@ -183,6 +185,54 @@ static const char *highlight_init_both[] = {
   "default link DiagnosticSignWarn DiagnosticWarn",
   "default link DiagnosticSignInfo DiagnosticInfo",
   "default link DiagnosticSignHint DiagnosticHint",
+
+  "default link @error Error",
```

Comment:
Do we really want this on by default ? I think that one of the things `nvim-treesitter` does is remove that because it is not nice.

Reply:
Yeah, this is used by nvim-treesitter for `ERROR` nodes. Maybe nvim-treesitter should use `@error.parser` for that instead (and highlight it as `NONE`).

Alternatively, use `@error.syntax` (or somesuch) for the (single) actual error node (in HTML) and map that here?

Reply:
Ignore that -- the default group is `ErrorMsg`, so `Error` should not show up.

Reply:
@clason You mean no action here, and instead adjust nvim-treesitter (which is going to require a bunch of adjustments anyway)?

Reply:
well, option 1 would need no adjustment here; option 2 would.

Not sure which is better? (I'd say option 1 sounds cleaner, but wouldn't help here due to the fallback.)

Reply:
Let's keep things like they are here -- it's nvim-treesitter that should be better about separating these; (Maybe make `ERROR` a `@tserror` instead and not map this by default -- @vigoux?)

Reply:
The mapping in itself makes sense, and maybe it is indeed up to the queries to avoid using this group too much.